### DISCUSSION TITLE
batch correction for multiple multiomic (GEX/ATAC) datasets

### DISCUSSION THREAD

**Discussion by Leonrunning** [stuart-lab/signac#1187] [Q&A] [answered]
Hi, I am looking for suggestions to correct batch effects between multiple multiomic (GEX/ATAC) datasets .

Should I separate the two assays and integrate (CCA) datasets of each assay separately before merging them together and calculating a WNN graph?

Or are there any other better practices to achieve this same goal? Thanks

**Comment by timoast** [comment, marked as answer]
Please see this discussion on the Seurat github page: https://github.com/satijalab/seurat/discussions/4015

**Reply by Leonrunning** [reply]
thanks, that is very helpful
